# BalanceOdm

# imu
float32   ax
float32   ay
float32   az

float32   gx
float32   gy
float32   gz

# enc
int16   left_enc
int16   right_enc

# condition for debug
int8   status_1
int8   status_2

# cmd_echo speed for debug
int16   left_speed
int16   right_speed
